packages/nimble-components/src/table-column/mapping/mappings/icon/tests/mapping-icon.stories.ts

```diff
@@ -0,0 +1,55 @@
+import { html } from '@microsoft/fast-element';
```

Comment:
Using the story seems to be causing runtime errors:
![iconcolumnsorting](https://github.com/ni/nimble/assets/1588923/fb7591d4-bde8-4bd7-b26c-51ad2449b265)